docusaurus/docs/dev-docs/database-migrations.md

```diff
@@ -74,3 +74,36 @@ module.exports = {
 ```
 
 </details>
+
+
+## Additional considerations
+
+### Using Strapi Instance for migrations
+
+If a user opts not to use Knex directly for migrations and instead utilizes the Strapi instance, it is important to wrap the migration code with `strapi.db.transaction()`. Failure to do so may result in migrations not rolling back if an error occurs.
```

Comment:
```suggestion
:::danger
If a user opts not to use Knex directly for migrations and instead utilizes the Strapi instance, it is important to wrap the migration code with `strapi.db.transaction()`. Failure to do so may result in migrations not rolling back if an error occurs.
:::
```

If I understand correctly, this might result in data loss so the highest-level of callout is required.